src/test/java/org/jenkinsci/plugins/workflow/multibranch/ReadTrustedStepTest.java

```diff
@@ -232,12 +232,13 @@ public void pathTraversalRejected() throws Exception {
 
         WorkflowRun b = p.getLastBuild();
         assertEquals(1, b.getNumber());
-        r.assertLogContains("secrets/master.key references a file that is not inside " + r.jenkins.getWorkspaceFor(p).getRemote(), b);
+        r.assertLogContains("master.key references a file that is not inside " + r.jenkins.getWorkspaceFor(p).getRemote(), b);
```

Comment:
I think this was just an issue with `secrets\master.key` vs `secrets/master.key`